Formula/xrootd.rb

```diff
@@ -4,7 +4,7 @@ class Xrootd < Formula
   url "https://xrootd.slac.stanford.edu/download/v4.12.5/xrootd-4.12.5.tar.gz"
   sha256 "164c83171e0bfc4d15ed55abde5f8b4d6413aa516c6b1faefabca0e6ba18275f"
   license "LGPL-3.0-or-later"
-  head "https://github.com/xrootd/xrootd.git"
+  head "https://github.com/xrootd/xrootd.git", branch: "stable-4.12.x"
```

Comment:
Why would we pin HEAD to a release branch?

Reply:
yeah, I was thinking something else the other day.

Yeah, we should use master branch for default head build.